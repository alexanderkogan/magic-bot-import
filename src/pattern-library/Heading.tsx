import * as React from 'react'
import { Props } from '../react-typings'

export default (props: Props) => <h1>{props.children}</h1>
