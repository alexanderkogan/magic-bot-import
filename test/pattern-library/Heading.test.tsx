import * as React from 'react'

import Heading from '../../src/pattern-library/Heading'
import { mount } from '../test-utils'

describe('Content', () => {
    it('renders', () => {
        const element = mount(<Heading>children</Heading>)
        expect(element).toMatchSnapshot()
    })
})
