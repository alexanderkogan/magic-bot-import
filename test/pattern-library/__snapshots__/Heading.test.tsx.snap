// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders 1`] = `
<Component>
  <h1>
    children
  </h1>
</Component>
`;
